src/constants/ticks.ts

```diff
@@ -0,0 +1,16 @@
+import JSBI from 'jsbi'
+
+export interface TickData {
+  tick: number
+  liquidityNet: string
+  liquidityGross: string
```

Comment:
i think we can delete all the `liquidityGross` logic as it doesn't seem like we use it anywhere?